### PR TITLE
fix: not able to update user prefs

### PR DESCRIPTION
**Pull request by safwanyp**
## What does this PR do?

This PR fixes an issue where _**User Preferences**_ could not be updated in the Appwrite console.

Fixes: https://github.com/appwrite/appwrite/issues/5282

## Test Plan

- Ran `npm test`
- Did manual testing for multiple scenarios of adding/deleting user preferences in the console.

## Related PRs and Issues

* https://github.com/appwrite/appwrite/issues/5282

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes. I ran the linter, formatter, and checks to ensure my code follows the same standard as the original code.

### CODE REVIEW

File: src/routes/console/project-[project]/auth/user-[user]/updatePrefs.svelte [hunk 1]
```diff
@@ -13,6 +13,8 @@
         if (JSON.stringify(prefs) !== JSON.stringify(Object.entries($user.prefs))) {
             if (!!prefs[prefs.length - 1][0] && !!prefs[prefs.length - 1][1]) {
                 arePrefsDisabled = false;
+            } else if (prefs[prefs.length - 1][0] === '' || prefs[prefs.length - 1][1] === '') {
+                arePrefsDisabled = false;
```
Comment (TorstenDittmann) [inline src/routes/console/project-[project]/auth/user-[user]/updatePrefs.svelte:17]: this allows me enter empty keys, not sure this is what should be allowed 🤔 

Bildschirm­foto 2023-03-29 um 15 10 12

Comment (safwanyp) [reply src/routes/console/project-[project]/auth/user-[user]/updatePrefs.svelte:17]: Whoops. Didn't acciunt for this. Will update soon.

**Comment by safwanyp** [comment]
@TorstenDittmann seems like it was a simple mistake I made while using OR instead of AND lol. Please have a look and let me know if you spot any more issues!

**Comment by safwanyp** [comment]
That's interesting. So we allow as many empty values as needed, but restrict the number of empty keys to 1?

**Comment by safwanyp** [comment]
Hey @TorstenDittmann 

> So we allow as many empty values as needed, but restrict the number of empty keys to 1?

Pinging you for this clarification ^

**Comment by stnguyen90** [comment]
> Hey @TorstenDittmann
> 
> > So we allow as many empty values as needed, but restrict the number of empty keys to 1?
> 
> Pinging you for this clarification ^

Definitely no more than 1 of the same key because you can't have 2 key-value pairs with the same key.

**Pull request closed without merging**